src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1825,17 +1825,9 @@ public void updateFeedbackQuestion(FeedbackQuestionAttributes updatedQuestion)
         feedbackQuestionsLogic.updateFeedbackQuestion(updatedQuestion);
     }
 
-    public void updateFeedbackQuestionWithResponseRateCheck(FeedbackQuestionAttributes updatedQuestion)
-            throws InvalidParametersException, EntityDoesNotExistException {
-
-        Assumption.assertNotNull(ERROR_NULL_PARAMETER, updatedQuestion);
-        feedbackQuestionsLogic.updateFeedbackQuestionWithResponseRateCheck(updatedQuestion);
-    }
-    
     /**
-     * Deletes the feedback session but not the questions and
-     * responses associated to it.
-     * Fails silently if no such feedback session. <br>
+     * Deletes the feedback question and the responses associated to it. Fails
+     * silently if there is no such feedback session. <br>
```

Comment:
@unyoungwax, oops, minor change here: <code>...if there is no such feedback session</code> should be <code>...if there is no such feedback question</code>